File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/TemplateDeliveryTransporterDetail.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class TemplateDeliveryTransporterDetail {
    @JsonProperty("ActualDriverStartDateTime")
    public String actualDriverStartDateTime;
    @JsonProperty("DriverLayoverLeg")
    public String driverLayoverLeg;
    @JsonProperty("DriverName")
    public String driverName;
    @JsonProperty("DriverOccupationalLicenseNumber")
    public String driverOccupationalLicenseNumber;
    @JsonProperty("DriverVehicleLicenseNumber")
    public String driverVehicleLicenseNumber;
    @JsonProperty("ShipmentDeliveryId")
    public Integer shipmentDeliveryId;
    @JsonProperty("VehicleLicensePlateNumber")
    public String vehicleLicensePlateNumber;
    @JsonProperty("VehicleMake")
    public String vehicleMake;
    @JsonProperty("VehicleModel")
    public String vehicleModel;
}

File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/template_delivery_transporter_detail.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class TemplateDeliveryTransporterDetail(TypedDict, total=False):
    ActualDriverStartDateTime: str
    DriverLayoverLeg: str
    DriverName: str
    DriverOccupationalLicenseNumber: str
    DriverVehicleLicenseNumber: str
    ShipmentDeliveryId: int
    VehicleLicensePlateNumber: str
    VehicleMake: str
    VehicleModel: str
